### DISCUSSION TITLE
Pages custom domain keeps disabling itself every few hours

### DISCUSSION THREAD

**Discussion by server-KaTys** [community/community#38352] [Other Feature Feedback, Questions, & Ideas]
Hello,

Since a few days ago, my [`server-KaTys/KaTys`](https://github.com/server-KaTys/KaTys) repo's Pages custom domain is disabling itself every few hours.

Here are the latest timestamps (provided by Uptime Robot) :
- at 2022-11-04 07:12:55 AM UTC+1, the custom subdomain disables itself ;
- at 2022-11-04 08:07:54 PM UTC+1, I enable it back ;
- at 2022-11-04 11:38:15 PM UTC+1, it disables itself again ;
- at 2022-11-04 11:43:14 PM UTC+1, I enable it back again ;
- at 2022-11-05 07:38:35 AM UTC+1, it disables itself again ;
- at 2022-11-05 12:46:46 PM UTC+1, I enable it back again.

Unless it's prevented, I'm pretty sure it's going to disable itself again in a few hours.

Thanks

---

This is a *machine user* (as per [GitHub Docs, section *Personal accounts*, § 4](https://docs.github.com/en/get-started/learning-about-github/types-of-github-accounts#personal-accounts)). Further replies will be submitted from @KaKi87.

**Comment by KaKi87** [comment]
And it just happened again, at 2022-11-05 03:53:32 PM UTC+1.

Should I enable it back ?

**Reply by yoannchaudet** [reply]
Hello,

The automation you use to publish your site is just discarding the `CNAME` file at the root of the repo. For Pages sites that builds from a branch, this file tells what your custom domain is.

Just make sure this file stays on your `master` branch and your custom domain won't disappear anymore.

Example: https://github.com/server-KaTys/KaTys/commit/67ec32d61f4a55c7f1e5a8e18be16426b3461e10

**Reply by KaKi87** [reply]
That was it, thanks !
